### ISSUE TITLE
0.4.0 release empty

### ISSUE THREAD

**Issue by elzibubble** [jgrodziski/talltale#4]
I think something went horribly wrong with the latest release:

```
% unzip -l 0.4.0/talltale-0.4.0.jar
Archive:  0.4.0/talltale-0.4.0.jar
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  2019-07-09 20:01   META-INF/
      138  2019-07-09 20:01   META-INF/MANIFEST.MF
        0  2019-04-12 13:35   META-INF/maven/
        0  2019-04-12 13:35   META-INF/maven/talltale/
        0  2019-04-12 13:35   META-INF/maven/talltale/talltale/
      143  2019-04-12 13:35   META-INF/maven/talltale/talltale/pom.properties
     3156  2019-07-09 20:01   META-INF/maven/talltale/talltale/pom.xml
---------                     -------
     3437                     7 files
```

**Comment by elzibubble** [comment]
Also the README still references 0.3.0

**Comment by elzibubble** [comment]
Thanks for turning those PRs around so fast :)

**Comment by jgrodziski** [comment]
Ok it's fixed with 0.4.3, I took the opportunity to switch to tools.deps and add proper CLJS tests runner and fix the time part that wasn't properly isomorphic.
